app/api/items/items.go

```diff
@@ -144,6 +144,9 @@ type itemChild struct {
 	// Defaults to true  if `can_edit` >= 'all_with_grant', false otherwise.
 	// It is always possible to set this field to the same or a lower value, `can_edit` doesn't matter in this case.
 	EditPropagation bool `json:"edit_propagation"`
+	// Can be set to true if `can_request_help_to` can be set (i.e. if user have “can_grant_view ≥ content” on item)
+	// Defaults to true if `can_request_help_to` can be set, false otherwise.
+	RequestHelpPropagation bool `json:"request_help_propagation"`
```

Comment:
The same `// It is always possible to set this field to the same or a lower value, `can_edit` doesn't matter in this case.` remark should apply here as well. -> If it is already "true", you can leave send "true" even if you do not have the perm to set it to true otherwise.

Reply:
fixed